codec/src/main/java/io/netty/handler/codec/compression/LzfEncoder.java

```diff
@@ -137,4 +138,29 @@ protected void encode(ChannelHandlerContext ctx, ByteBuf in, ByteBuf out) throws
             recycler.releaseInputBuffer(input);
         }
     }
+
+    @Override
+    public boolean isClosed() {
+        return false;
+    }
+
+    @Override
+    public ChannelFuture close() {
+        return ctx().newFailedFuture(new UnsupportedOperationException());
+    }
+
+    @Override
+    public ChannelFuture close(ChannelPromise promise) {
+        return promise.setFailure(new UnsupportedOperationException());
+    }
+
+    @Override
+    public void close(ChannelHandlerContext ctx, ChannelPromise promise) throws Exception {
+        ctx.close(promise);
+    }
+
+    @Override
+    protected ChannelFuture finishEncode(ChannelHandlerContext ctx, ChannelPromise promise) {
+        return promise.setFailure(new UnsupportedOperationException());
```

Comment:
Instead of rejecting the close request, we could simply close the channel immediately without scheduling any write timeout, etc?